src/test/kotlin/com/exactpro/th2/codec/json/TestUtils.kt

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Please, specify a package in the file. Also, probably we should create a sub-package `util`